### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by cadop** [andybak/CGALUnity#1]
The links provided in description are broken.  Is this project (or the original) working at all?

**Comment by andybak** [comment]
Yeah. I'd cloned the original repo before it was removed so that's all I managed to preserve. Maybe @Scrawk can help?

**Comment by Scrawk** [comment]
I found CGAL a pain to use so I gave up on it. The project files are long gone.

Everything that project does can be found in other projects written in C#.

There's [triangle dot net](https://archive.codeplex.com/?p=triangle) for constrained triangulations.

There's [clipper](http://www.angusj.com/delphi/clipper.php) for polygon boolean and offset operations.

There's [MIConvexHull](https://github.com/DesignEngrLab/MIConvexHull) for any dimension hull and triangulations.
